torch_geometric/data/batch.py

```diff
@@ -183,3 +184,65 @@ def num_graphs(self) -> int:
     def __reduce__(self):
         state = self.__dict__.copy()
         return DynamicInheritanceGetter(), self.__class__.__bases__, state
+
+    def subbatch(self, idx):
+        # convert index to mask
+        if isinstance(idx, slice) or \
+           (isinstance(idx, Tensor) and idx.dtype == torch.long) or \
+           (isinstance(idx, np.ndarray) and idx.dtype == np.int64) or \
+           (isinstance(idx, Sequence) and not isinstance(idx, str)):
+            mask = torch.zeros(self.num_graphs, dtype=torch.bool)
```

Comment:
```suggestion
            mask = index_to_mask(index, self.num_graphs)
```
this function is in `torch_geometric.util`

Reply:
`index_to_mask` only accepts a `Tensor`, while here we must consider indexes of other types. Maybe `index_to_mask` should consider other types as well?

Reply:
Good point. I agree it could accept other types